packages/jest-worker/src/__tests__/process-integration.test.ts

```diff
@@ -183,33 +184,30 @@ describe('Jest Worker Integration', () => {
   });
 
   it('should check for memory limits', async () => {
```

Comment:
@phawxby Could I ask to take a look why is this test failing now?

The test file was converted into TypeScript and I had to refactor `mockBuildForkedProcess` helper into `MockChildProcess` class. Reverting this change make the test pass again. Strange. All other tests work as before. Is it just some magic number issue or I broke something?

Reply:
Sorry about the noise. This was my mistake. Fixed it –  https://github.com/facebook/jest/pull/13416/commits/380b65dd89418ae40a132af7bf0c684562ff54b7